Index: squeeze-packaging/squeeze
===================================================================
--- squeeze-packaging.orig/squeeze	2011-08-19 00:13:39.078901011 +0200
+++ squeeze-packaging/squeeze	2011-08-19 00:20:39.798901001 +0200
@@ -16,6 +16,7 @@
 
 # select alternate sound
 SFX="default"
+WHERE="/usr/share/games/glgames"
 
-cd "$(dirname $0)"
-ruby -rubygems ./lib/tools/launcher.rb squeeze -f $FULLSCREEN -x $WINX -y $WINY -g $GFX -s $SFX $@
+cd "$WHERE"
+ruby -rubygems "$WHERE/lib/tools/launcher.rb" squeeze -f $FULLSCREEN -x $WINX -y $WINY -g $GFX -s $SFX $@
